_posts/2023-03-11-release.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Release 2023.10
+author: Priya
+date: 2023-03-11
+category: Jekyll
+layout: post
+---
+
+### Enhancements
+
+#### Revamped BigQuery Connector for Better User Experience
+* Now, users can remove and edit a BigQuery service file in the export section. Also, multiple service files can be added.
+* Enhanced warning system: If a user creates an export that will overwrite data already exported from another view, a warning message will appear on validation. The user will be asked to confirm if this is intentional before proceeding further.
+
+
+### Defect fixes
+
+* Resolved an issue where the BigQuery table name was not displayed in error notifications.
+* Addresses an issue in the Connectors Modal where the list of columns was not followed by the data type in the drop-down of unique sequence columns.
```

Comment:
Fixed an issue in the dropdown for the Unique sequence column in the Connectors modal where the names of the column in the list were missing column type suffixes.

Reply:
done